src/main/java/com/fundynamic/d2tm/game/behaviors/SimpleSelectLogic.java

```diff
@@ -0,0 +1,24 @@
+package com.fundynamic.d2tm.game.behaviors;
+
+/**
+ *
+ * A very straightforward select logic using a boolean flag to determine if something is selected or not.
+ *
+ */
+public class SimpleSelectLogic implements Selectable {
```

Comment:
units actually have a different 'selected' animation. So I kind of prepared for that here. While the logic remains the same, the drawing is different.